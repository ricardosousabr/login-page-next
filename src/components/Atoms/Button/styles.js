import styled from 'styled-components'
import colors from '../../../tokens/colors.js'
import screens from '../../../tokens/screen.js'

export const ButtonForm = styled.button`
  background: ${colors.primary};
  border: none;
  color: white;
  width: 190px;
  height: 30px;
  border-radius: 5px;

  @media (min-width: ${screens.small}) {
    width: 220px;
  }
`
