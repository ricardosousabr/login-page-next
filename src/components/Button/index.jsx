import { ButtonForm } from './styles'

export default function Button(props) {
  return (
    <>
      <ButtonForm>{props.children}</ButtonForm>
    </>
  )
}
