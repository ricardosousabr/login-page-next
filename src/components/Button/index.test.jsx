/**
 * @jest-environment jsdom
 */

import React from 'react'
import { render } from '@testing-library/react'
import Button from './index'

it('renders homepage unchanged', () => {
  const { container } = render(<Button />)
  expect(container).toMatchSnapshot()
})
