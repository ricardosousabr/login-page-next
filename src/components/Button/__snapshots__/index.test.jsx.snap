// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <button
    class="styles__ButtonForm-sc-1penh9l-0 kKfXFj"
  />
</div>
`;
